Makefile

```diff
@@ -9,8 +9,8 @@ update-version:
 clean:
 	  (./code/gradlew -p code clean)
 
-checkstyle: core-checkstyle
-		
+checkstyle: core-checkstyle signal-checkstyle lifecycle-checkstyle identity-checkstyle
```

Comment:
Just to confirm, we intend to exclude compatibility right?

Reply:
Yes. The plan is to remove compatibility before release from our repo unless any extension uses it.